x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/select_logs.tsx

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable @elastic/eui/href-or-on-click */
+
+import React, { PropsWithChildren, MouseEvent } from 'react';
+import {
+  EuiTitle,
+  EuiLink,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiSpacer,
+  EuiCard,
+  EuiIcon,
+  EuiIconProps,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import {
+  StepPanel,
+  StepPanelContent,
+  StepPanelFooter,
+} from '../../../shared/step_panel';
+import { useWizard } from '.';
+import { useKibanaNavigation } from '../../../../hooks/use_kibana_navigation';
+
+export function SelectLogs() {
+  const { navigateToKibanaUrl, navigateToAppUrl } = useKibanaNavigation();
+  const { goToStep, getState, setState } = useWizard();
+
+  function onBack() {
+    navigateToKibanaUrl('/app/observabilityOnboarding/customLogs');
+  }
+
+  return (
+    <StepPanel
+      title={i18n.translate(
+        'xpack.observability_onboarding.selectLogs.chooseType',
+        {
+          defaultMessage: 'Choose what logs to collect',
+        }
+      )}
+    >
+      <StepPanelContent>
+        <LogsTypeSection
+          title={i18n.translate(
+            'xpack.observability_onboarding.selectLogs.customLogs',
+            {
+              defaultMessage: 'Custom logs',
+            }
+          )}
+        >
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false} style={{ width: '50%' }}>
+              <OptionCard
+                title={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.streamLogFiles',
+                  {
+                    defaultMessage: 'Stream log files',
+                  }
+                )}
+                iconType="desktop"
+                onClick={() => {
+                  setState({ ...getState(), logsType: 'log-file' });
```

Comment:
i'm not 100% sure if we should update the state before continuing to the next step, or if the next step will be specific for streaming log files. Like maybe the fact that they are in `configureLogs` step already implies they clicked this option. We can leave it in for now, but we may want to clean it up later if it's not necessary.

Reply:
Totally agree, but I think we will discover more about it as we advance in the topic.